mysql-test/t/audit_log_many_connections.test

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
sorry - missed it. For upstream submission can you convert the testcase to audit_null? Please also add --source include/have_audit_null... at the top